docs/02-concepts.md

```diff
@@ -1,3 +1,19 @@
 # Concepts in the AXLearn Library
 
 **This doc is still under construction.**
+
+
+## Input Batch Sharding
+
+WWhen using `SpmdTrainer`, it is common to read and process inputs across all processes and hosts. For the most common use case where you want each process to have an equal portion of the input batch, this process is mostly transparent to the user. For more complex use cases, it can be helpful to have a general idea of the what is happening behind the scenes.
```

Comment:
```suggestion
When using `SpmdTrainer`, it is common to read and process inputs across all processes and hosts. 
For the most common use case where you want each process to have an equal portion of the input batch, this process is mostly transparent to the user. 
For more complex use cases, it can be helpful to have a general idea of the what is happening behind the scenes.
```

Reply:
Done.